pkg/maintainer/spv/spv.go

```diff
@@ -0,0 +1,68 @@
+package spv
+
+import (
+	"context"
+	"time"
+
+	"github.com/ipfs/go-log/v2"
+
+	"github.com/keep-network/keep-core/pkg/bitcoin"
+)
+
+var logger = log.Logger("keep-maintainer-spv")
+
+func Initialize(
+	ctx context.Context,
+	config Config,
+	chain Chain,
+	btcChain bitcoin.Chain,
+) {
+	spvMaintainer := &spvMaintainer{
+		config:   config,
+		chain:    chain,
+		btcChain: btcChain,
+	}
+
+	go spvMaintainer.startControlLoop(ctx)
+}
+
+type spvMaintainer struct {
+	config   Config
+	chain    Chain
+	btcChain bitcoin.Chain
+}
+
+func (sm *spvMaintainer) startControlLoop(ctx context.Context) {
+	logger.Info("starting SPV maintainer")
+
+	defer func() {
+		logger.Info("stopping SPV maintainer")
+	}()
+
+	for {
+		err := sm.maintainSpv(ctx)
+		if err != nil {
+			logger.Errorf(
+				"error while maintaining SPV: [%v]; restarting maintainer",
+				err,
+			)
+		}
+
+		select {
+		case <-time.After(sm.config.RestartBackOffTime):
+		case <-ctx.Done():
+			return
+		}
+	}
+}
+
+func (sm *spvMaintainer) maintainSpv(ctx context.Context) error {
+	logger.Infof("Maintaining SPV proof...")
+	logger.Infof("spv chain: ", sm.chain)
+	logger.Infof("btc chain:", sm.btcChain)
```

Comment:
This is missing `%v` so it will log weird lines. I propose we get rid of those two logs completely.

```
2023-06-22T17:35:14.999+0200    INFO    keep-maintainer-spv     spv/spv.go:61   spv chain: %!(EXTRA *ethereum.TbtcChain=&{0x14000c088c0 0x14002184180 0x14002546100 0x14002807000 0x140028c3800 0x1400293b580})
2023-06-22T17:35:14.999+0200    INFO    keep-maintainer-spv     spv/spv.go:62   btc chain:%!(EXTRA *electrum.Connection=&{0x140000400d8 0x1400037ccb0 0x14000124090 {wss://electrum.boar.network:2083 10000000000 60000000000 30000000000 120000000000 300000000000}})
```

Reply:
Removed the logs completely in 0c4522c1ab29d6d86f9c47ef77cb0e3ca24eaec0.